zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -359,11 +359,11 @@ private void prepareSource(HeliumPackage pkg, String[] moduleNameVersion,
         .append(" from \"../" + mainFileName + "\"\n");
 
     loadJsRegister.append(HELIUM_BUNDLES_VAR + ".push({\n");
-    loadJsRegister.append("id: \"" + moduleNameVersion[0] + "\",\n");
-    loadJsRegister.append("name: \"" + pkg.getName() + "\",\n");
-    loadJsRegister.append("icon: " + gson.toJson(pkg.getIcon()) + ",\n");
-    loadJsRegister.append("type: \"" + pkg.getType() + "\",\n");
-    loadJsRegister.append("class: " + className + "\n");
+    loadJsRegister.append("  id: \"" + moduleNameVersion[0] + "\",\n");
```

Comment:
My suggestion is to use the following style so that no string concatenation is performed.
```suggestion
    loadJsRegister.append(idention).append("id: \"").append(moduleNameVersion[0]).apend("\",\n");
```

Of course, the idention variable must be defined before, but it also makes it easier to change it globally. It also makes the code more readable, explaining why two spaces are inserted.  I don't think these are necessary from a purely programmatic point of view, do they?

Reply:
Thanks for pointing that out. I've removed the indentation (as it didn't really improve readability) and replaced the string concatenation with `.append()`, following your suggestion.